$NetBSD: patch-mi,v 1.5 2013/05/12 01:49:16 makoto Exp $

Add DragonFly support.

 # Allow the user to ignore the OS_VERSION, which is usually irrelevant.
--- config/config.mk.orig	2013-05-07 18:01:19.000000000 +0900
+++ config/config.mk	2013-05-11 22:32:56.000000000 +0900
@@ -103,7 +103,7 @@
 # but save the version to allow multiple versions of the same base
 # platform to be built in the same tree.
 #
-ifneq (,$(filter FreeBSD HP-UX Linux NetBSD OpenBSD SunOS,$(OS_ARCH)))
+ifneq (,$(filter DragonFly FreeBSD HP-UX Linux NetBSD OpenBSD SunOS,$(OS_ARCH)))
 OS_RELEASE	:= $(basename $(OS_RELEASE))
 
 # Allow the user to ignore the OS_VERSION, which is usually irrelevant.
